carma-messenger-ui/website/scripts/control.js

```diff
@@ -1,577 +1,577 @@
+    newInput.setAttribute('title', checkboxTitle);
+
+    var newLabel = document.createElement('label');
+    newLabel.id = 'lbl' + checkboxId;
+    newLabel.htmlFor = newInput.id;
+
+    if (isRequired == true)
+        newLabel.innerHTML = '<span style="color:#FF0000">* </span>'
+
+    newLabel.innerHTML += checkboxTitle;
+
+
+    container.appendChild(newInput);
+    container.appendChild(newLabel);
+
+}
+/*
+* Adds a new checkbox onto the container.
+*/
+function createBtnWithLabelElement(container, btnId, btnValue,afterClickBtnValue,  btnName,labelTitle,IsHide,  btnClickfunc) {
+
+    var alreadyExists = document.getElementById(btnId);
+
+    if (alreadyExists != null)
+        return;
+
+    var divBtnLabelUnit=document.createElement('div');
+    divBtnLabelUnit.className="divBtnLabelUnit"+btnId;
+    var newInput = document.createElement('input');
+    newInput.type = 'button';
+    newInput.name = btnName;
+    newInput.id = btnId;
+    newInput.setAttribute('title', btnValue);
+    newInput.value=btnValue;   
+
+    var newLabel = document.createElement('label');
+    newLabel.id = 'lbl' + btnId;
+    newLabel.htmlFor = newInput.id;    
+    newLabel.innerHTML += labelTitle;
+    
+    //hide it during the first creation
+    if(IsHide){
+        newLabel.style.display='none';
+        newInput.style.display='none';
+    }
+    divBtnLabelUnit.appendChild(newLabel);
+    divBtnLabelUnit.appendChild(newInput);
+   
+
+    //container.appendChild(divBtnLabelUnit);
+    return divBtnLabelUnit;
+
+}
+
+
+/**** create a html <div>
+ *                      <div>
+ *                            <div><div><span></span></div>
+ *                            <div></div>
+ *                      </div>
+ *                  </div> tags  
+ * */
+function createDivFramePanelText(divFrameId,divFrameCss,divPanelId,divPanelCss,smallTitle,divTextId,divTextCSS,divTextValue)
+{
+    var alreadyExists = document.getElementById(divFrameId);
+
+    if (alreadyExists != null)
+        return;
+
+    var divFrame=document.createElement('div');
+    divFrame.className=divFrameCss;
+    divFrame.id=divFrameId;
+
+    var divPanel = document.createElement('div');
+    divPanel.className=divPanelCss;
+    divPanel.id=divPanelId;
+
+    var smalllDiv = document.createElement("div");
+    smalllDiv.className="smallTextDiv";
+    var smallSpan = document.createElement("span");
+    var smallText = document.createTextNode(smallTitle);
+    smallSpan.appendChild(smallText);
+    smalllDiv.appendChild(smallSpan);
+
+    var divText = document.createElement('div');
+    divText.id=divTextId;    
+    divText.className=divTextCSS;
+    if(divTextValue!=null && divTextValue!="undefined" && divTextValue.length > 0)
+    { 
+        if(divTextValue.includes("../../images/")){
+            var img = document.createElement('img');
+            //image path
+            img.src=divTextValue; 
+            img.alt="Image is not availabe";
```

Comment:
still incorrect. Needs the letter L after the letter b. Should say "Image is not available."